app/helpers/general-date.js

```diff
@@ -0,0 +1,9 @@
+import Helper from '@ember/component/helper';
+import moment from 'moment';
+
+export function headerDate(params) {
+  let timezone = params[1] ? params[1] : moment.tz.guess();
```

Comment:
Deconstruct params

Reply:
Shouldn't this rule only run on single allocation like
```js
let timezone = params[1]
``` 
in case of array

Reply:
@iamareebjamal Please update me on this rule, I couldn't find what sort of change I need to make for deconstruct params